Makefile

```diff
@@ -20,8 +20,6 @@ OUT_DIR=$(REPO_ROOT)/_output
 ################################################################################
 # ========================= Setup Go With Gimme ================================
 # go version to use for build etc.
-# setup correct go version with gimme
-PATH:=$(shell . hack/build/setup-go.sh && echo "$${PATH}")
 # go1.9+ can autodetect GOROOT, but if some other tool sets it ...
 GOROOT:=
 # enable modules
```

Comment:
We can remove this whole section now, right? The env should be handled by setup-go.sh.

Reply:
yeah, done